packages/backend/src/open_payments/wallet_address/service.ts

```diff
@@ -39,16 +39,15 @@ export type WalletAddressAdditionalPropertyInput = Pick<
 >
 
 export interface CreateOptions extends Options {
+  tenantId?: string
```

Comment:
Where might this be called where `tenantId` is unavailable to be passed into the service call?

Reply:
By not forcing the tenant-id, the `operatorTenantId` will be used from the config. 
Would it be better for us to just force the `tenantId` @njlie ? 

```nodejs
const tenantId = options.tenantId
      ? options.tenantId
      : deps.config.operatorTenantId
```